SimpleSetupGuide.md

```diff
@@ -84,6 +78,23 @@ See [this guide](Verify-REMS-Integration-Prototype-Works.md) to generate a test
 
 ## Cleanup and Useful Options
 
+### Obtain [Value Set Authority Center (VSAC)](https://vsac.nlm.nih.gov/) API key
```

Comment:
these sections maybe should be before the Run section I think, at least the section on the VSAC key especially. I noticed before we were linking to the dev set up guide, I'm fine with having it inline instead. Also the numbering above is not consistent as we go from 1. System Requirements to 2. Run. to 4. Verify everything is working. We are missing 3 entirely, but moving the VSAC and env stuff up should fix that ordering.

Reply:
Given that the VSAC Key is optional, I think putting it in the main workflow adds some unnecessary complexity for someone that just wants the thing to run. As of my last test, lack of a VSAC key no longer produces any errors.